plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -57,6 +57,8 @@ public class BigQueryConfig
     private Duration viewsCacheTtl = new Duration(15, MINUTES);
     private Duration serviceCacheTtl = new Duration(3, MINUTES);
     private Duration metadataCacheTtl = new Duration(0, MILLISECONDS);
+    @Deprecated
+    private boolean isLegacyMetadataListing;
```

Comment:
I didn't document `bigquery.legacy-metadata-listing` intentionally because I want to remove this property eventually.

Reply:
Should this be a `HiddenConfig` then?